packages/astro/src/@types/astro.ts

```diff
@@ -849,6 +849,29 @@ export interface AstroUserConfig {
 		 * ```
 		 */
 		split?: boolean;
+
+		/**
+		 * @docs
+		 * @name build.excludeMiddleware
+		 * @type {boolean}
+		 * @default {false}
+		 * @version 2.8.0
+		 * @description
+		 * Defines how the SSR code should be bundled when built.
```

Comment:
```suggestion
		 * Defines whether or not any SSR middleware code will be bundled when built.
```
Since this is a boolean, I think the initial description should present exactly what the choice is as a "whether or not." Also, since this refers specifically to middleware code, that should be somewhere in the sentence. This is just an example of what that sentence could look like. Please edit so that it's correct/has the right nuance!